### ISSUE TITLE
Max retries exceeded with url:

### ISSUE THREAD

**Issue by licheng5625** [bimmerconnected/bimmer_connected#144]
HTTPSConnectionPool(host='b2vapi.bmwgroup.com', port=443): Max retries exceeded with url: /gcdm/oauth/token (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7235f5d0>: Failed to establish a new connection: [Errno 111] Connection refused'))
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3/connection.py", line 157, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/connection.py", line 84, in create_connection
    raise err
  File "/usr/local/lib/python3.7/site-packages/urllib3/util/connection.py", line 74, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused

What is the maxmal number of retries ? I did not retrie the data too often. Is it the possible to limit the frenquence ?

Thank you

**Comment by rikroe** [comment]
Apparently BMW is experiencing some major hickups...
The [ConnectedDrive website](https://www.bmw-connecteddrive.de/app/index.html#/portal/dashboard) and the Android app are unavailable as well.

**Comment by WhimsySpoon** [comment]
Can confirm this issue affecting me in the UK also.

Connections to `http://b2vapi.bmwgroup.com/` fail, whereas connections on the US version (`https://b2vapi.bmwgroup.us/`) are still being answered, sort of.

I had a similar issue when I purchased my vehicle earlier in the year. I spent an hour or two on the forecourt with various people from the dealership trying to work out why we couldn't register the vehicle/use the app etc. It was only when the dealership's expert returned from lunch did he tell us that they update the platforms on a Saturday and it doesn't always go to plan. I confirmed this when I telephoned the ConnectedDrive helpline and a recent message on the IVR stated that there were issues. It sorted itself out a day or so later.

Hopefully this issue is related to server downtime at BMW's end and doesn't require any massive changes to the component.

Given that the `.us` address seems fine, is there anyone in the US having issues at the moment, or is just just the rest of the world (outside of China)?
